### PR TITLE
Add Cloud Vision landmark detection sample.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: vision/landmarkDetection.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+// [START app]
+// [START import_libraries]
+var gcloud = require('gcloud');
+// [END import_libraries]
+
+// [START authenticate]
+// You must set the GOOGLE_APPLICATION_CREDENTIALS and GCLOUD_PROJECT
+// environment variables to run this sample. See:
+// https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/docs/authentication.md
+var projectId = process.env.GCLOUD_PROJECT;
```
Comment (jonparrott) [inline vision/landmarkDetection.js]: Sidenode: The Python client library detects this automatically now.

**Comment by jonparrott** [comment]
LGTM

**Comment by jerjou** [comment]
LGTM

**Pull request merged**